scripts/build-server

```diff
@@ -12,11 +12,13 @@ RKE_VERSION="$(grep -m1 'github.com/rancher/rke' go.mod | awk '{print $2}')"
 
 # Inject Setting values
 DEFAULT_VALUES="{\"rke-version\":\"${RKE_VERSION}\"}"
+RANCHER_PRIME="${RANCHER_PRIME:-false}"
```

Comment:
We won't be able to do this at build time of the go binary. So I believe this needs to go in env var.

Reply:
It can't be an env var, because then it would be changeable at run time. It has to be done at build time. Why can't we have drone set this at build time?